AlphaWalletTests/Extensions/UIColorExtensionTests.swift

```diff
@@ -41,8 +41,8 @@ class UIColorExtensionTests: XCTestCase {
     }
 
     func testLightDarkColorMode() throws {
-        let lightColor = Colors.appRed
-        let darkColor = Colors.appWhite
+        let lightColor = UIColor.red
```

Comment:
Why does the original version not work? It seems like it should work with any pair of 2 different colors?

Reply:
How about a different pair of arbitrary RGB colors that is not any of the semantic colors used in the app that passes? Eg 50, 100, 150 and another one? Helps show that it doesn’t rely on any style semantics

Reply:
Ok I used 1,0,0,1 for red and 1,1,1,1 for white.